docs/html-support.md

```diff
@@ -0,0 +1,121 @@
+# Enable HTML Templates in EmailOTP
+
+Use this configurations if you need to use HTML Templates in the Emails.
+**This feature is supported from Identity Server v5.6 onwards**.
+
+1. Configure Email settings in \<IS_HOME>/repository/conf/output-event-adapters.xml file. 
```

Comment:
Can we say:
Open the <IS_HOME>/repository/conf/output-event-adapters.xml file and configure the appropriate email settings as follows:

Reply:
Fixed in https://github.com/wso2-extensions/identity-outbound-auth-email-otp/pull/30/commits/f578717bad3139812d57d434f1d4275141d260be